analytics/management/commands/update_analytics_counts.py

```diff
@@ -36,6 +40,19 @@ def add_arguments(self, parser):
 
     def handle(self, *args, **options):
         # type: (*Any, **Any) -> None
+        LOCK_DIR = "/tmp/update_analytics_lock"
```

Comment:
We should move this value to be a setting set to `var/update_analytics_counts_lock` in DEVELOPMENT and `/home/zulip/update_analytics_counts_lock` in production.

Otherwise, this looks good!